config/settings.py

```
@@ -151,17 +177,31 @@
+
+    # BYTE_PATROL_ALLOWED_IPS = _get_env_list(
+    #     "BYTE_PATROL_ALLOWED_IPS", ["127.0.0.1", "69.255.206.86"])
+    BYTE_PATROL_ALLOWED_IPS = ["76.85.43.21"]
+
+    BYTE_PATROL_PROTECTED_PATHS = _get_env_list(
+        "BYTE_PATROL_PROTECTED_PATHS",
+        [r"^/admin/.*$", r"^/checkup/?.*$", r"^/byte_patrol/protected/.*$"],
+    )
+    BYTE_PATROL_CACHE_TIMEOUT = 9000
+    BYTE_PATROL_FORBIDDEN_RESPONSE = HttpResponseForbidden(
+        "Access Denied: Your IP address is not allowed."
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)